.github/workflows/push-jsp-on-tag.yml

```diff
@@ -12,6 +12,15 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
+    - uses: actions/checkout@v2
+    - name: Check out CLDR
+      uses: actions/checkout@v2
+      with:
+        repository: unicode-org/cldr
+        path: cldr
+    - name: Backup Unicodetools and CLDR for jsps
```

Comment:
dumb question: Why do two workflows need to do the same backup?

Reply:
because they both attempt to build the Docker image.  This one is only run on tag, the other is run on every commit.

Reply:
i had neglected the build-jsp.yml at first which is why the build failed before